mage_ai/services/k8s/config.py

```diff
@@ -18,12 +20,21 @@ class K8sResourceConfig(BaseConfig):
 
 @dataclass
 class K8sExecutorConfig(BaseConfig):
+    config_file: str = CONFIG_FILE
     container_config: Dict = None
     job_name_prefix: str = None
-    namespace: str = DEFAULT_NAMESPACE
+    metadata: Dict = None
+    container: Dict = None
```

Comment:
This looks same with `container_config`.
https://github.com/mage-ai/mage-ai/blob/master/mage_ai/services/k8s/job_manager.py#L110-L111
Have you tested with it? It supports all the [container config keys](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1Container.md)